frontend/src/InputForm.js

```diff
@@ -364,20 +364,23 @@ class InputForm extends React.Component {
 
     submitState.criterion.timePreferences = submitState.criterion.timePreferences.map((times) => 
         this.timeToTimeRange(0, times.startTime, times.endTime));
-    var json = JSON.stringify(submitState);
-    //! This will be changed to send the json to the backend
-    console.log(json);
-  } 
+    fetch("/handleUserInput", {
+      method:"POST",
+      headers: {'Content-Type':'application/json'},
+      body: JSON.stringify(submitState)
+    }).then(response => response.json())
+      .then(scheduleList => console.log(JSON.stringify(scheduleList)));
```

Comment:
Should the `scheduleList` be stored anywhere so the frontend can read from it/render the schedules? This looks like placeholder code at the moment, but storing the variable at this point would still be better than logging it.